firebase-database-collection/firebase-database-collection.gradle

```diff
@@ -14,13 +14,41 @@
 
 apply plugin: 'com.android.library'
 
-def androidVersion = 26
+def androidVersion = 28
 
 android {
     compileSdkVersion androidVersion
+    defaultConfig {
+        minSdkVersion 14
+    }
+    buildTypes {
```

Comment:
These lines look a little suspicious:

- Do we need the empty `sourceSets`?
- Shouldn't the `compileOptions` be inherited?
- I don't think we want `useProguard true` anymore, we're not proguarding these SDKs going forward.  Or is that only needed to pass forward the `consumerProguardFiles`?

Reply:
- `compileOptions` is not inherited. It always failed when I didn't set it to Java 8.
- I only follow pattern of other projects for the use of `useProguard true`. So should we remove it, as well in other projects?

Reply:
@ashwinraghav ?